runtime/parachains/src/runtime_api_impl/v1.rs

```diff
@@ -231,6 +232,29 @@ pub fn session_index_for_child<T: initializer::Config>() -> SessionIndex {
 	<shared::Module<T>>::session_index()
 }
 
+/// Implementation for the `AuthorityDiscoveryApi::authorities()` function of the runtime API.
+/// Gets current and historical authority ids using session_info module.
+pub fn get_current_and_historical_authority_ids<T: initializer::Config>() -> Vec<AuthorityDiscoveryId> {
+	let current_session_index = session_index_for_child::<T>();
+	let earliest_stored_session = <session_info::Module<T>>::earliest_stored_session();
+	let mut authority_ids = vec![];
```

Comment:
yeah, we need to include authorities for the next session as well here
the rationale is outlined in https://github.com/paritytech/substrate/pull/6788